com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalTarget.cs

```diff
@@ -475,6 +475,7 @@ static class CoreRenderStates
             { RenderState.ZWrite(ZWrite.On) },
             { RenderState.Cull(Cull.Back), new FieldCondition(Fields.DoubleSided, false) },
             { RenderState.Cull(Cull.Off), new FieldCondition(Fields.DoubleSided, true) },
+            { RenderState.ZClip(ZClip.On) }, //, new FieldCondition(UniversalFields.ZClip, false) },
```

Comment:
I wonder how platforms that doesn't support a feature a handled in the graph?

Remove commented code.